doc-site/docs/architecture/distributed_sequencer.md

```diff
@@ -0,0 +1,1437 @@
+
+From the perspective of the request sender, only one request response should be consumed and all other should be ignored.  If an error is received and the reason for the error is a transient one, there is no guarantee that sending the same request (matching request id) in future will result in a different response.  The retry protocol is intended to mitigate unreliability at the network transport layer ( i.e. when no response is received).  To retry in the case of other transient error's, a new request (i.e. a message with a different request id) must be initiated. 
+
+```mermaid
+sequenceDiagram
+  %%{init: { "sequence": { "mirrorActors":false }}}%%
+  participant S as Request Sender
+  participant R as Request Receiver
+  loop Every RetryInterval
+    S -) R: Request
+  end
+  alt request is valid
+    R --) S: Response
+  else
+    R --) S: Error
+  end
+```
+
+### Message specification and handler responsibilities
+
+The following concrete message exchanges defines the responsibilities and expectation of each node in the network.
+
+
+
+#### <a name="message-transaction-delegation-command"></a>Transaction delegation
+
+
+This is an instance of the `Command` pattern where the `command sender` is the sender node for the given transaction and the `command receiver` is the node that has been chosen, by the `sender` as the `coordinator`.  The `sender` sends a `DelegationCommand` message...
+
+```mermaid
+sequenceDiagram
+  %%{init: { "sequence": { "mirrorActors":false }}}%%
+  participant S as Sender
+  participant C as Coordinator
+  loop Every RetryInterval
+    S -) C: DelegationCommand
+  end
+  alt delegation valid
+    C --) S: DelegationAccepted
+    C -->> C: Coordinate Transaction
+    activate C
+    deactivate C
+  else
+    C --) S: DelegationRejected
+  end
+  
+```
+
+
+##### <a name="message-transaction-delegation-command"></a>Delegation Command
+```proto
+message DelegationCommand {
+    string transaction_id = 1;
+    string delegate_node_id = 2;
+    string delegation_id = 3; //this is used to correlate the acknowledgement back to the delegation. unlike the transport message id / correlation id, this is not unique across retries
+    bytes private_transaction = 4; //json serialized copy of the in-memory private transaction object
+    int64 block_height = 5; // the block height upon which this delegation was calculated (the highest delegation wins when crossing in the post)
+}
+```
+
+As a member of the coordinator committee, a node that receives a `DelegationCommand` must either respond by sending `DelegationAccepted` or `DelegationRejected` message.  
```

Comment:
`must either` seems incorrect here. `should either` seems more correct, as it's perfectly expected in our protocol (in my understanding) that a coordinator might just restart and forget in this situation.

Reply:
This might be wrong though as I think about it - maybe I'm looking for a definition of what we mean by `must`